### PR TITLE
[WIP][DOC][3915] Add Getting Started with PacHub.

### PR DESCRIPTION
**Pull request by svekars**
- Added the Getting Started section.
- Added screenshots.
- Added the Tutorials section.
- Added the Getting Started screencast

### CODE REVIEW

**Comment by svekars** [comment]
Work in progress. Need feedback from @gabrielgrant on a few things.

File: doc/pachub/pachub_getting_started.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+   ```bash
+   $ pachctl version --client-only
+   1.9.5
+   ```
+
+1. Configure a Pachyderm context:
+
+   1. In the Pachyderm Hub UI, click **Details** next to your cluster.
+   1. Copy the IP address and protocol of your cluster.
```
[the remark this answers is not in the document]
Comment (JoeyZwicker) [reply doc/pachub/pachub_getting_started.md]: I think a screenshot of the details view of a cluster would be good here. Hub team is adding the OTP info into the "details" view so wait for that to be included.

File: doc/pachub/pachub_getting_started.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
+# Getting Started with Pachyderm Hub
```
Comment (JoeyZwicker) [inline doc/pachub/pachub_getting_started.md]: Not sure if or where to add it, but might be good to say "Pachyderm Hub is in beta and we'd love your feedback and help you troubleshoot any issues or confusion. Please join our public slack channel."

I know its already 10 other places, but we especially want to make sure Hub users get funneled there so we can get feedback.

File: doc/pachub/pachub_getting_started.md [hunk 3]
```diff
@@ -162,16 +166,17 @@ To connect to your cluster, complete the following steps:
 1. Log in to your Pachyderm cluster:
 
    1. Go to the Pachyderm Hub UI.
-   1. In the list of clusters, click **Dashboard**.
-      Pachyderm Hub opens the Dashboard in a new window.
-   1. Click **Settings**.
-   1. Click **Generate New One-Time Password Now**.
+   1. In the list of clusters, click **Details**.
+   1. Click **>**.
```
Comment (JoeyZwicker) [inline doc/pachub/pachub_getting_started.md]: one minor part of the PR connor is working on changes this. It's a very easy update for me to make myself if/when it lands. This is fine for now

**Pull request merged**